baseapp/baseapp.go

```diff
@@ -688,6 +689,17 @@ func (app *BaseApp) runTx(mode runTxMode, txBytes []byte) (gInfo sdk.GasInfo, re
 	// Result if any single message fails or does not have a registered Handler.
 	result, err = app.runMsgs(runMsgCtx, msgs, mode)
 	if err == nil && mode == runTxModeDeliver {
+		// Run optional postHandlers
+		if app.postHandler != nil {
+			newCtx, err := app.postHandler(runMsgCtx, tx, mode == runTxModeSimulate)
```

Comment:
```suggestion
			newCtx, err := app.postHandler(runMsgCtx, tx, false)
```

Reply:
Actually, I think we should run postHandler on simulate (like we do run anteHandler on simulate).

Instead of your suggestion, I did [46e2102](https://github.com/cosmos/cosmos-sdk/pull/11985/commits/46e21028a546d740d7b7a52760345755638e0d24), wdyt?

Reply:
Sure.